From fdf9bd229bedb1683d18b8beaf2ca787034bb78b Mon Sep 17 00:00:00 2001
From: Gregor Riepl <onitake@gmail.com>
Date: Sat, 8 Jul 2023 14:39:13 +0200
Subject: [PATCH] Replace size_t format string in InfillTest.cpp

This will fix an incorrect format causing test
failure on 32-bit architectures.
---
 tests/InfillTest.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/InfillTest.cpp b/tests/InfillTest.cpp
index 23b083f5..b8d034a0 100644
--- a/tests/InfillTest.cpp
+++ b/tests/InfillTest.cpp
@@ -100,7 +100,7 @@ namespace cura
             result_lines(result_lines),
             result_polygons(result_polygons)
         {
-            name = makeName("InfillTestParameters_P%d_Z%d_C%d_L%lld__%lld", (int)params.pattern, (int)params.zig_zagify, (int)params.connect_polygons, params.line_distance, test_polygon_id);
+            name = makeName("InfillTestParameters_P%d_Z%d_C%d_L%lld__%zu", (int)params.pattern, (int)params.zig_zagify, (int)params.connect_polygons, params.line_distance, test_polygon_id);
         }
 
         friend std::ostream& operator<<(std::ostream& os, const InfillTestParameters& params)
-- 
2.40.1

